src/Core/Framework/Api/ApiDefinition/Generator/Schema/AdminApi/components/schemas/consentListResponse.json

```
@@ -0,0 +1,45 @@
+{
+  "openapi": "3.1.0",
+  "info": [],
+  "paths": [],
+  "components": {
+    "schemas": {
+      "consentListResponse": {
+        "type": "array",
+        "description": "List of consent states for the authenticated user",
+        "items": {
+          "$ref": "#\/components\/schemas\/consentState"
+        }
+      },
+      "consentState": {
```

most of the component names are uppercased, so I'd stay that way
```suggestion
      "ConsentState": {
```